.github/workflows/kind_multicluster_e2e_tests.yaml

```diff
@@ -0,0 +1,22 @@
+name: Run kind e2e multi-cluster tests
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: [ main ]
```

Comment:
Do we dare to run on PRs without security? Perhaps we should limit self-hosted to main merges only until we have some security checks?

Reply:
Considering we will probably have plenty of PRs for which we will want to run multi-cluster tests I would like to run them on PRs. With that said, security is certainly a big concern. What about manually kicking off the workflow? If the PR is from a maintainer, I think we would be fine with running.

Reply:
Yes, certainly a possibility. Just something to prevent automated run of unknown PRs not from our maintainers.